chord_metadata_service/discovery/pydantic_models.py

```diff
@@ -3,7 +3,7 @@
 from bento_lib.discovery import FieldDefinition, OverviewSection, DiscoveryEntity, SearchSection
 from pydantic import BaseModel, Field, RootModel
 from rest_framework.request import Request as DrfRequest
-from typing import TypeAlias
+from typing import TypeAlias, Optional, Dict
```

Comment:
this is old-fashioned type hinting (Dict from typing and Optional) - just use `dict[...]` and union types (`a | b`)